test/inspect-test.js

```diff
@@ -90,24 +90,39 @@ tape('v8 inspect', (t) => {
         -1,
         'cons string content');
 
-    if (process.version >= 'v5.0.0')
-      return sess.send(`v8 inspect ${fn}`);
-
-    // No Context debugging for older node.js
-    sess.quit();
-    t.end();
+    sess.send(`v8 inspect -s ${fn}`);
   });
 
-  sess.linesUntil(/}>/, (lines) => {
+  sess.linesUntil(/^>/, (lines) => {
     lines = lines.join('\n');
-    t.ok(/\(previous\)/.test(lines), 'method.previous');
-    t.ok(/scopedVar[^\n]+"scoped value"/.test(lines), 'method.scopedValue');
-
-    let match = lines.match(
-        /\(closure\)=(0x[0-9a-f]+)[^\n]+function: closure/i);
-    t.ok(match, '`method` should have `closure`');
 
-    sess.send(`v8 inspect ${match[1]}`);
+    // Include 'source:' and '>' to act as boundaries. (Avoid
+    // passing if the whole file it displayed instead of just
+    // the function we want.)
+    const methodSource = "  source:\n" +
+    "  Class.prototype.method = function method() {\n" +
```

Comment:
It turns out the start position of the function source code in the script is the beginning of the arguments list at "(" not the function keyword or the (optional) function name. That makes sense but left the output looking a bit odd so I've inserted the function keyword and the function name (if there is one) ahead of the arguments list for readability.
The output expected by the test is now:

```
    const methodSource = "  source:\n" +
    "function method() {\n" +
    "    throw new Error('Uncaught');\n" +
    "  }\n" +
    ">"
```